common/src/balance_unit.rs

```diff
@@ -179,6 +180,29 @@ impl BalanceUnit {
         };
         Ok(Self::new(balance, self.is_divisible))
     }
+
+    pub fn into_indivisible(mut self, mode: RoundMode) -> Self {
```

Comment:
this function is not needed after changing the `fill_order_book`. I think I'll remove it soon, since there's no demand